Updated images on environmental variable page

nataliefiann opened the pull request:
## What are you changing in this pull request and why?

I've updated the images on the environmental variable page as part of the UI project

I've updated the images under the following sections:

- Overriding environment variables at the job level (the first image)
- Overriding environment variables at the personal level (the three images under this section)


## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages
-->

<!-- vercel-deployment-preview -->
---
🚀 Deployment available! Here are the direct links to the updated files:


- https://docs-getdbt-com-git-nfiann-img-sweep-dbt-labs.vercel.app/docs/build/environment-variables

<!-- end-vercel-deployment-preview -->

nataliefiann committed:
Updated images on environmental variable page

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

mirnawong1 reviewed (approved):
thanks @nataliefiann , am i right in thinking it's only the ide refresh that's been changed here and the gif removed? approved 

<img width="783" alt="Screenshot 2024-12-10 at 11 53 58" src="https://github.com/user-attachments/assets/336a92e9-8781-47a2-8a6f-fcae3c012f21">

nataliefiann commented:
PR description:

I've updated the images on the environmental variable page as part of the UI project

I've updated the images under the following sections:

- Overriding environment variables at the job level (the first image)
- Overriding environment variables at the personal level (the three images under this section)

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep
